app/models/candidate.rb

```
@@ -13,15 +13,16 @@ class Candidate < ApplicationRecord
   has_many :application_forms
 
   def current_application
-    choice = ApplicationChoice.first_or_create do |ac|
+    application_form = application_forms.first_or_create
```

Should this call `first_or_create!` for failing fast?